src/Core/SemanticVersion.cs

```diff
@@ -85,18 +108,27 @@ public string SpecialVersion
             private set;
         }
 
+        /// <summary>
+        /// SemVer 2.0.0 metadata. This is not used for comparing or sorting.
+        /// </summary>
+        public string Metadata

```

Comment:
Yes, it could be called that instead. NuGet.Versioning uses Metadata so I've added the same field here to match.